### ISSUE TITLE
EntryNotFound (FileSystemError) after applying vscode v1.60.1 on MacOS ('#' in file path)

### ISSUE THREAD

**Issue by bheimbaugh** [DavidAnson/vscode-markdownlint#191]
After applying the latest vscode update v1.60.1 on MacOS, the Markdownlint extension v0.44.0 started logging the following errors to console for the file `/Users/brookh/dev/Nettools Security/Issues/road-map#3347/road-map#3347.md`

```
ERROR: Exception while linting with markdownlint-cli2:
EntryNotFound (FileSystemError): Unable to resolve non-existing file '/Users/brookh/dev/Nettools Security/Issues/road-map/#3347/road-map/#3347.md'
	at _handleError (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:160087)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

The exception text shows extraneous `/`'s before the `#` characters in `.../road-map#3347/road-map#3347.md`. I assume this is the cause of the problem.

**Comment by DavidAnson** [comment]
Thanks for the specific details! VS Code stores paths as URLs and there may be an inconsistency with how the extension deals with that. Other paths work fine, but # is always broken like this, it seems?

**Comment by bheimbaugh** [comment]
It's been working up until the v1.60.1 this morning, but I had to disable the plugin because all my files are in paths like that.

**Comment by DavidAnson** [comment]
Sorry for the trouble! I've just published version 0.44.1 which is identical to the previous version 0.43.2. I will investigate a couple of bug reports and re-release the new version with fixes soon.
